nuclear_simulator/data_gen/config_engine/composers/comprehensive_composer.py

```
@@ -0,0 +1,417 @@
+            # Create primary_system section if it doesn't exist
+            if 'primary_system' not in config:
+                config['primary_system'] = {}
+            
+            # Move parameters from top level to primary_system
+            for param_name, param_value in top_level_primary_params.items():
+                config['primary_system'][param_name] = param_value
+                # Remove from top level
+                del config[param_name]
+                print(f"   � Moved {param_name} to primary_system")
+            
+            print(f"   ✅ Primary system restructuring complete")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)